src/open_inwoner/cms/cases/views/status.py

```diff
@@ -505,9 +505,14 @@ def handle_document_upload(self, request, form):
                 document_type.informatieobjecttype_url,
                 source_organization,
             )
-            created_relationship = connect_case_with_document(
-                self.case.url, getattr(created_document, "url", None)
-            )
+
+            try:
+                created_relationship = connect_case_with_document(
+                    self.case.url, created_document.get("url")
+                )
+            except AttributeError:
```

Comment:
@pi-sigma I've added the AttributeError handling here because created_document could be `None`, however I think we might want to separate the error handling for the both fields only create the relationship if the document is actually created.

This might requires some small refactoring here.

Reply:
@svenvandescheur I don't think we need to expose this to the user; they only care about whether the upload worked or not.

Relatedly, I tested the code again and it now fails with `getattr` for me too, as it should. So I think we're good to go.